src/js/components/Layout/v2/Subsection.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+
+import PropTypes from 'prop-types';
+import { RiArrowDownSLine, RiArrowUpSLine } from 'react-icons/ri';
+
+import Translate from 'utils/Translate';
+
+const Subsection = ({
+  title,
+  collapsable,
+  children,
+}) => {
+  // If a subsection is not collapsable, it is always expanded
+  // (collapsable: false --> expanded: true)
+  // If a subsection is collapsable, it is not expanded by default
+  // (collapsable: true --> expanded: false)
+  const [expanded, setExpanded] = useState(!collapsable);
+
+  const triggerCollapse = () => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <div className="v2-subsection">
+      <div className="subsection-title-wrapper">
+        <span
+          role="button"
+          tabIndex={0}
+          onClick={collapsable ? () => triggerCollapse() : null}
+          onKeyDown={collapsable ? () => triggerCollapse() : null}
+          style={collapsable ? { cursor: 'pointer' } : { cursor: 'unset' }}
+        >
+          <Translate id={title.label} defaultMessage={title.defaultMessage} />
+          {collapsable
+            && (expanded
+              ? <RiArrowUpSLine className="arrow-up" />
+              : <RiArrowDownSLine className="arrow-down" />)}
+        </span>
+      </div>
+      <div className={`subsection-body ${expanded ? 'subsection-body-expanded' : ''}`}>
+        {children}
+      </div>
+    </div>
```

Comment:
Maybe it is possible to reduce the amount of JS code inside the HTML...
Maybe you can create an object:
```
{
collapsable: {
onClick: ....,
onKeyDown: ...,
style: ...,
},
...
}
```
and spread it on this span?
I am just thinking out loud, you can skip it